openreview/conference/builder.py

```diff
@@ -1069,12 +1069,14 @@ def setup_final_deadline_stage(self, force=False, hide_fields=[]):
         if self.use_area_chairs:
             self.set_area_chairs()
 
-    def setup_post_submission_stage(self, force=False, hide_fields=[]):
+    def setup_post_submission_stage(self, force=False, hide_fields=None):
```

Comment:
is it possible to remove `hide_fields` from here? I think the value should always come from the submission stage object.

Reply:
I've left this because there are a bunch of tests that directly use this instead of setting the submission stage.